backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -635,6 +644,73 @@ describe("Listings", () => {
     })
   })
 
+  describe("Create listing", () => {
+    it("should create a new listing", async () => {
+      const adminAccessToken = await getUserAccessToken(app, "admin@example.com", "abcdef")
+
+      // A listing requires a jurisdiction, so we just pick the first one we can get
+      const jurisdictionRes = await supertest(app.getHttpServer())
+        .get(`/jurisdictions`)
+        .set(...setAuthorization(adminAccessToken))
+        .expect(200)
+      const jurisdictionId = jurisdictionRes.body[0].id
+
+      const listing: ListingPublishedCreateDto = {
```

Comment:
nit: what do you think about adding a TODO to import the minimal-listing.json instead of filling in all these fields here? This would require a little futzing with the imported thing (since minimal-listing.json conforms to a `ListingImport` (not a `ListingPublishedCreateDto`), but it would prevent minimal-listing.json from falling out of sync (so long as this test is run in presubmits).

Reply:
Started actually just using `minimal-listing.json` directly here, but there was a little too much difference between it and `ListingPublishedCreateDto`, so I left a TODO instead.